translate/canon/ListLanguageNames.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright(c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START translate_list_language_names]
+
+using Google.Cloud.Translation.V2;
+using System;
+using System.Collections.Generic;
+
+public partial class TranslateSample
+{
+    public IList<Language> ListLanguageNames()
+    {
+        TranslationClient client = TranslationClient.Create();
+        IList<Language> languages = client.ListLanguages(
+            "en");  // The language to use to list the language names.
```

Comment:
As an alternative to putting a comment on top of the code line, you might consider using named parameters C# feature, which according to documentation from [here](https://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Translation.V2/api/Google.Cloud.Translation.V2.TranslationClient.html#Google_Cloud_Translation_V2_TranslationClient_ListLanguages_System_String_System_Nullable_Google_Cloud_Translation_V2_TranslationModel__) would be:

`client.ListLanguages(target: "en");`

Also, I see that nuget's documentation also includes some of the snippets. I suggest we add a link to that so folks can see more samples(based on autogenerated code) if they want.

Reply:
Yeah, I thought about that too.  I chose comments because we can get more descriptive in comments.  We can add details that are not manifest in the parameter name.